test/test_da.py

```diff
@@ -0,0 +1,67 @@
+
+
+import ot
+import numpy as np
+
+# import pytest
+
+
+def test_OTDA():
+
+    n = 150  # nb bins
+
+    xs, ys = ot.datasets.get_data_classif('3gauss', n)
+    xt, yt = ot.datasets.get_data_classif('3gauss2', n)
```

Comment:
OK we should open an issue and handle that in a separate PR I think, this one is mainly for testing